kafka-handler/src/java/org/apache/hadoop/hive/kafka/KafkaDagCredentialSupplier.java

```diff
@@ -123,6 +122,9 @@ private Token<?> getKafkaDelegationTokenForBrokers(Configuration conf, TableDesc
     config.put(SaslConfigs.SASL_JAAS_CONFIG, jaasConfig);
 
     LOG.debug("Jaas config for requesting kafka credentials: {}", jaasConfig);
+    Configuration copy = new Configuration(conf);
```

Comment:
This variable name could be changed. Copy is too generic, ex: config, copyconfig or tableconfig.

Reply:
Fixed in https://github.com/apache/hive/pull/4430/commits/cfb464b7dd9c3b1af9425b61e856082f388f84a9